### ISSUE TITLE
Pretrained Indoor Model

### ISSUE THREAD

**Issue by toshi2k2** [DQiaole/ZITS_inpainting#50]
Hi, 
Can you upload the pretrained Indoor data model - the results of which you share in your paper? Also, can you share the trained models of the comparative methods you show results for in your paper?

Thank you.

**Comment by DQiaole** [comment]
Hi,
Sorry for the late reply, our model has lost as our server is broken last month.

**Comment by toshi2k2** [comment]
Thanks for the reply. Can you share the script to train the model for the Indoor data from scratch?
Additionally, can you clarify a few things -
1. Are the wireframes and edgemaps masked after detection on the entire image?
2. Is the TSR trained only on the Indoor training data?
3. Is the HAWP model - which is used to get the wireframes - trained on a superset of the Indoor data?

**Comment by DQiaole** [comment]
Hi,
For the script that trains a model on Indoor, I will do my best to find it as soon as possible. There should be little different from the scripts for Places2.
1. sure, please see the following codes:
https://github.com/DQiaole/ZITS_inpainting/blob/6fd5fc0518eac201032ea1d90fd71310b9050a77/src/utils.py#L267-L276
2. sure.
3. According to MST, the HAWP model is re-trained on ShanghaiTech with mask augmentation. And ShanghaiTech is a subset of our Indoor dataset. Please refer to MST for more details.
